modules/lib-bash/activation-init.sh

```diff
@@ -88,6 +88,24 @@ function setupVars() {
     fi
 }
 
+function checkUsername() {
+  local expectedUser="$1"
+
+  if [[ $USER != $expectedUser ]]; then
```

Comment:
```suggestion
  if [[ $USER != "$expectedUser" ]]; then
```

RHS of `!=` is subject to globbing

Reply:
Thanks 🙂